talks/from-chat-to-agent/styles/index.css

```
@@ -0,0 +1,592 @@
+.speaker-profile--compact .speaker-profile__avatar img { border: 0; }
+.speaker-profile--compact .speaker-profile__eyebrow { display: none; }
+.speaker-profile--compact .speaker-profile__name { display: none; }
+.speaker-profile--compact .speaker-profile__links { flex-direction: column; gap: 2px; margin-top: 0; }
+.speaker-profile--compact .speaker-profile__links a { display: flex; align-items: center; gap: 5px; font-size: 14px; }
+.speaker-profile--compact .speaker-profile__link-label { display: none; }
+.speaker-profile__icon { width: 11px; height: 11px; opacity: .72; }
+
+@keyframes profile-spin { to { transform: rotate(360deg); } }
+
+.section-label { display: flex; align-items: center; gap: 14px; font-family: 'DM Mono', monospace; font-size: 14px; letter-spacing: .12em; color: rgba(255,255,255,0.9); }
+.section-label__line { width: 42px; height: 1px; background: currentColor; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)